src/mail_room.py

```diff
@@ -0,0 +1,166 @@
+#  _*_ coding:utf-8 _*_
+"""A simple python program to keep track of names and donations."""
+from __future__ import print_function
+import sys
+import io
+
+
+GIFTER_DICT = {}
+
+prompts = ["Would you like to 'Send a Thank you' or 'Create a Report'?", "Enter the donator name or press 'L' to see a list of previous names", "How much is the donation?"]
+email_text = "Dear {},\nThank you so much for you donation of ${}. You are a good person.\n"
+
+
+def main_controller():
+    """Control flow of program."""
+    begin = send_or_report()
+    if begin is True:
+        name = get_name()
+        check_dict(name, GIFTER_DICT)
+        ammt = get_ammt()
+        update_dict(name, ammt)
+        send_email(name, ammt)
+        main_controller()
+    elif begin is False:
+        to_print = create_report()
+        if to_print is False:
+            main_controller()
+        else:
+            print_report(to_print)
+            write_to_file(to_print)
+            main_controller()
+
+
+def prompt_usr(prompt):
+    """Return the user response to a prompt."""
+    try:
+        input = raw_input
+    except NameError:
+        pass
+    usr_answer = input('{} \n>>>>>> '.format(prompt))
```

Comment:
Have you tried to run this since updating it this way?  There's a fatal error here that is preventing this function from being able to work.

The problem is that you attempt to alias `input` to `raw_input` *inside* your function definition.  What this does is create a *local* symbol `input` that shadows the builtin input.  So when you reach this line (40) and attempt to use it in Python 3 it is unbound and causes an `UnboundLocalError`.  

The solution is to move lines 36-39 to the module scope so that the aliasing happens there.  Then you can *use* `input` in this function without needing to worry which version it is.